src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final String name = "circle";

```

Comment:
first, if this is a constant - it should be named in UPPER_CASE
second, let's better remove this field from all figure classes